README.md

```diff
@@ -69,6 +69,16 @@ Using bidi, path-exists? would look like:
   (boolean (bidi/match-route app-routes path)))
 ```
 
+By default, clicking a link to the currently active route will not run the
```

Comment:
I'd prefer "trigger" to "run" both here and on line 73.

Reply:
Thanks for the feedback - I've addressed each comment in the above two commits